src/Raven.Server/Config/Categories/SecurityConfiguration.cs

```diff
@@ -10,7 +10,12 @@ namespace Raven.Server.Config.Categories
 {
     public class SecurityConfiguration : ConfigurationCategory
     {
-        [Description("The path to .pfx certificate file. If specified, RavenDB will use HTTPS/SSL for all network activities. You can use the '~/' prefix to refer to RavenDB's base directory.")]
+        [Description("The .pfx certificate in Base64 format. If specified, RavenDB will use HTTPS/SSL for all network activities. Certificate setting priority order: 1)Base64 2)Executable 3)Path")]
```

Comment:
Priority probably better as: 
* base64
* path
* exec

The reason is the level of complexity / security offered.